content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -504,6 +518,16 @@ Without this amendment, the owner reserve for a SignerList ranges from 15 to 50
 With this amendment enabled, the owner reserve for a new SignerList is 5 XRP, regardless of the number of signers. The reserve requirement for previously-created SignerList objects remains unchanged. To reduce the reserve requirement of SignerList objects created before this amendment was enabled, use a [SignerListSet transaction](signerlistset.html) to replace the SignerList after this amendment has been enabled. (The replacement can be identical to the previous version.)
 
 
+## NegativeUNL
```

Comment:
We should not document at this time. There's no corresponding amendment for it at this time.

Reply:
Aha, I looked at [the wrong `-b8` PR](https://github.com/ripple/rippled/pull/3459). The [right `-b8` PR](https://github.com/ripple/rippled/pull/3483) doesn't list it. My source code crawler found [the commented-out amendment](https://github.com/ripple/rippled/blob/7b048b423e8ae08a54018a89231d050b9f562855/src/ripple/protocol/impl/Feature.cpp#L135) and added it to the list of things "in development".

Maybe it's a good idea to leave this documented but change the "introduced" version to TBD?

Reply:
>Maybe it's a good idea to leave this documented but change the "introduced" version to TBD?

I think that's _probably_ fine.